metrics_utility/base/collector.py

```diff
@@ -169,50 +171,59 @@ def delete_tarballs(self):
     #
     def _calculate_collection_interval(self, since, until):
         _now = now()
+        original_since = since
+        original_until = until
+        self.logger.info(f'Original since-until: {original_since} to {original_until}')
 
         # Make sure that the endpoints are not in the future.
         if until is not None and until > _now:
             until = _now
-            self.logger.warning(f'End of the collection interval is in the future, setting to {_now}.')
+            self.logger.info(f'End of the collection interval is in the future, setting to {_now}.')
```

Comment:
these should still be warnings, right?

When we override what the user has set, that's a warning, not an info message